Source: http://flex.cvs.sourceforge.net/viewvc/flex/flex/gen.c?r1=2.97&r2=2.98&pathrev=HEAD
Upstream: Yes
Reason: The code produced generates warnings, n is compared against max_size that is of type
	size_t
Index: flex/gen.c
diff -u flex/gen.c:2.97 flex/gen.c:2.98
--- flex/gen.c:2.97	Thu May 31 06:21:57 2007
+++ flex/gen.c	Thu Apr 10 18:08:47 2008
@@ -1890,7 +1890,7 @@
 			outn ("\tif ( YY_CURRENT_BUFFER_LVALUE->yy_is_interactive ) \\");
 			outn ("\t\t{ \\");
 			outn ("\t\tint c = '*'; \\");
-			outn ("\t\tint n; \\");
+			outn ("\t\tsize_t n; \\");
 			outn ("\t\tfor ( n = 0; n < max_size && \\");
 			outn ("\t\t\t     (c = getc( yyin )) != EOF && c != '\\n'; ++n ) \\");
 			outn ("\t\t\tbuf[n] = (char) c; \\");
